src/libraries/System.Runtime.InteropServices/tests/ComInterfaceGenerator.Tests/RcwAroundCcwTests.cs

```diff
@@ -386,5 +386,39 @@ public void IStringArrayMarshallingFails_Failing()
                 obj.ByValueOutParam(strings);
             });
         }
+
+        [Fact]
+        public unsafe void IHideWorksAsExpected()
+        {
+            IHide obj = CreateWrapper<HideBaseMethods, IHide3>();
+
+            // IHide.SameMethod should be index 3
+            Assert.Equal(3, obj.SameMethod());
+            Assert.Equal(4, obj.DifferentMethod());
+
+            IHide2 obj2 = (IHide2)obj;
+
+            // IHide2.SameMethod should be index 7
```

Comment:
I assume you mean 5 and not 7?

Reply:
Yep, copy+paste error.

```suggestion
            // IHide2.SameMethod should be index 5
```